tests/functional/binary-cache.sh

```diff
@@ -46,28 +46,28 @@ basicDownloadTests() {
 
     nix-env --substituters "file://$cacheDir" -f dependencies.nix -qas \* | grep -- "---"
 
-    nix-store --substituters "file://$cacheDir" --no-require-sigs -r $outPath
+    nix-store --substituters "file://$cacheDir" --no-require-sigs -r "$outPath"
 
-    [ -x $outPath/program ]
+    [ -x "$outPath"/program ]
```

Comment:
same here, I usually quote the entire string unless there is a wildcard asterix.

I will now stop commenting this for all other occurrences.